api/dataprocessor_test.go

```diff
@@ -611,7 +611,10 @@ func TestAlignRequests(t *testing.T) {
 				reqRaw("a", 0, 30, 800, 10, consolidation.Avg),
 				reqRaw("b", 0, 30, 800, 60, consolidation.Avg),
 			},
-			[]mdata.AggSetting{},
+			mdata.AggSettings{
+				35 * 24 * 60 * 60,
```

Comment:
right now all these 35d ttl settings are basically ignored right? like if we changed them, the tests would still succeed? is this the case for all tests where this change is made?
I guess this puts us on track if we later want to make alignRequests smarter and take ttl into account. is that the purpose of these changes or do you have something else in mind? I'm having some trouble understanding what the goal is here.